fbc-integration-test/catalog-source/release/managed-workspace/release_strategy.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: ReleaseStrategy
+metadata:
+  name: fbc-strategy
+  namespace: managed-release-team-tenant
+spec:
+  pipeline: fbc-release
+  bundle: quay.io/scoheb/pipeline-fbc-release:latest
```

Comment:
Is this right? We want to use Scott's stuff instead of our production ones?

Reply:
this all irrelevant and removed, since we are running on RHTAP stage so ReleaseStrategy should be deployed on Stage backend